opencsp/app/fixed_pattern_deflectometry/lib/FixedPatternSetupCalibrate.py

```diff
@@ -108,6 +108,7 @@ def __init__(
             np.ndarray((x_max - x_min + 1, y_max - y_min + 1, 3)) * np.nan
         )
         self._num_dots: int
+        self._marker_corner_idxs: list[ndarray] = []
```

Comment:
If you want to be really exact about your types, you can specify what is in the ndarray like this:

```python
import numpy.typing as npt

self._marker_corner_idxs: list[npt.NDarray[np.int_]] = []
```

To be clear: I'm definitely not saying do this right now. More of an FYI, as you deem it useful/necessary.